osu.Game/Screens/Multi/Multiplayer.cs

```diff
@@ -129,12 +128,18 @@ protected Multiplayer()
                         }
                     },
                     new Header(screenStack),
-                    createButton = new CreateRoomButton
+                    createButton = CreateNewMultiplayerGameButton().With(button =>
                     {
-                        Anchor = Anchor.TopRight,
-                        Origin = Anchor.TopRight,
-                        Action = () => OpenNewRoom()
-                    },
+                        button.Anchor = Anchor.TopRight;
```

Comment:
not 100% sure about having this much layout specific code here rather than in a base class, but let's go with it for now.